src/lib/pod.ts

```diff
@@ -0,0 +1,33 @@
+import { Session } from '@inrupt/solid-client-authn-node';
+import { getEssServiceURI, EssServices } from '../config';
+
+export async function createProfileAndPod(session: Session) {
+  if (session.info.webId) {
+    await session.fetch(session.info.webId, {method: 'POST'})
+    const provision = await (await session.fetch(getEssServiceURI(EssServices.Provision), {method: 'POST'})).json()
+
+    await session.fetch(
+      `${session.info.webId}/provision`,
+      {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
```

Comment:
I've asked internally in Inrupt if this Content-Type shouldn't be `application/ld+json` instead (as clearly the payload if JSON-LD), but this is correctly following our current docs, so approving!